<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressource.qrc">
    <normaloff>:/images/icone.png</normaloff>:/images/icone.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	color: rgb(210, 255, 212);
}
QMenuBar{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 40, 5, 255), stop:1 rgba(0, 0, 0, 255));
}
QMenuBar::item {
	spacing: 1em;
	background: transparent;
	color: rgb(210, 255, 212);
	padding: 0.5ex 1em;
 }
QMenuBar::item:selected, QMenu::item:selected{
	background-color: rgb(0, 85, 0);
}
QMenu{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 40, 5, 255), stop:1 rgba(0, 0, 0, 255));
	color: rgb(210, 255, 212);
}
#action_quitter{
	background-color: red;
}
#centralwidget{
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.7, fx:0.5, 		fy:0.5, stop:0 rgba(0, 140, 18, 255), stop:1 rgba(0, 70, 9, 255));
}
#statusbar{
	background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0.003, y2:0.0023, stop:0 rgba(0, 38, 0, 255), stop:1 rgba(0, 26, 0, 255));
	color: white
}
QDockWidget *{
	background-color: rgb(0, 32, 0);
	color: rgb(210, 255, 212);
}
QDockWidget::title{
	text-align: center;
	background-color:black;
 }
QDockWidget::separator, QMainWindow::separator{
	background-color: black;
}
QDockWidget::separator:hover, QMainWindow::separator:hover{
	background-color: rgb(0, 32, 0);
}
QListView{
	border: 1px solid rgb(210, 255, 212);
	alternate-background-color: rgb(0, 64, 0);
	show-decoration-selected: 1;
}
QListView::item{
	padding-left: 1em;
	background-color: rgb(0, 32, 0);
}
QListView::item:selected{
	padding-left: 1em;
	background-color: rgb(0, 96, 0);
}
QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 94, 12, 255), stop:1 rgba(0, 29, 3, 255));
	border: 1px solid rgb(210, 255, 212);
	border-radius: 5px;
	padding: 0.5ex 1em;
}
QPushButton:hover{
	background-color: rgb(210, 255, 212);
	color: rgb(0, 32, 0);
}
QPushButton:pressed{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(0, 94, 12, 255), stop:1 rgba(0, 29, 3, 255));
	color: rgb(0, 32, 0);
	color: rgb(210, 255, 212);
}
:disabled{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 94, 12, 64), stop:1 rgba(0, 29, 3, 64));
	border: 1px solid rgba(210, 255, 212, 64);
	color: rgba(210, 255, 212, 64);
	padding: 0.5ex 1em;
}
QTextEdit, QLineEdit{
	border: 1px solid rgb(210, 255, 212);
}
QTabBar{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 40, 5, 255), stop:1 rgba(0, 0, 0, 255));
}
QTabBar::tab{
	background-color: rgb(0, 32, 0);
	border: 1px solid rgb(210, 255, 212);
	padding: 0.5ex 0.5em;
}
QTabBar::tab:hover, QTabBar::tab:selected:hover{
	background-color: rgb(210, 255, 212);
	color: rgb(0, 32, 0);
}
QTabBar::tab:selected{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 128, 0, 255), stop:1 rgba(0, 64, 0, 255));
	color: rgb(210, 255, 212);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>39</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuServeur">
    <property name="title">
     <string>&amp;Serveur</string>
    </property>
    <addaction name="action_connexion"/>
    <addaction name="action_solo"/>
    <addaction name="action_heberger"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>&amp;Aide</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTarot_virtuel">
    <property name="title">
     <string>&amp;Tarot virtuel</string>
    </property>
    <addaction name="action_quitter"/>
   </widget>
   <widget class="QMenu" name="menuFen_tres">
    <property name="title">
     <string>&amp;Fenêtres</string>
    </property>
    <addaction name="action_win_listejoueurs"/>
    <addaction name="action_win_discussion"/>
    <addaction name="action_win_connexion"/>
   </widget>
   <addaction name="menuTarot_virtuel"/>
   <addaction name="menuServeur"/>
   <addaction name="menuFen_tres"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dock_discussion">
   <property name="windowTitle">
    <string>Discussion</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="cont_boutons" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="bouton_envoyer">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Envoyer</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dock_connexion">
   <property name="windowTitle">
    <string>Connexion</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Adresse du serveur</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="champ_adresse"/>
        </item>
        <item>
         <widget class="QPushButton" name="bouton_changer_serveur">
          <property name="styleSheet">
           <string notr="true">:checked{
	border: 1px solid rgb(210, 255, 212);
	border-radius: 5px;
	padding: 0.5ex 1em;
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(0, 94, 12, 255), stop:1 rgba(0, 29, 3, 255));
	color: rgb(210, 255, 212);
}
:hover{
	background-color: rgb(210, 255, 212);
	color: rgb(0, 32, 0);
}
:pressed{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(0, 94, 12, 255), stop:1 rgba(0, 29, 3, 255));
	color: rgb(210, 255, 212);
}
:disabled{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 94, 12, 64), stop:1 rgba(0, 29, 3, 64));
	border: 1px solid rgba(210, 255, 212, 64);
	color: rgba(210, 255, 212, 64);
	padding: 0.5ex 1em;
}
* {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(94, 0, 12, 255), stop:1 rgba(29, 0, 3, 255));
	border: 1px solid rgb(255, 210, 212);
	color: red;
	border-radius: 5px;
	padding: 0.5ex 1em;
}</string>
          </property>
          <property name="text">
           <string>Ok</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="conteneur_identification" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Nom d'utilisateur</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="champ_nom"/>
        </item>
        <item>
         <widget class="QPushButton" name="bouton_connexion">
          <property name="text">
           <string>Se connecter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bouton_deconnexion">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Se déconnecter</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dock_liste_joueurs">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Liste des joueurs</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dock_lj_content">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="tarotv::ui::vue_jhj" name="liste_jhj">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_inviter">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Inviter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_connexion">
   <property name="text">
    <string>Se connecter à un serveur en ligne...</string>
   </property>
  </action>
  <action name="action_solo">
   <property name="text">
    <string>Jouer tout seul...</string>
   </property>
  </action>
  <action name="action_heberger">
   <property name="text">
    <string>Créer un serveur...</string>
   </property>
  </action>
  <action name="action_quitter">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_win_listejoueurs">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Afficher la liste des joueurs</string>
   </property>
  </action>
  <action name="action_win_discussion">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Afficher la fenêtre de discussion</string>
   </property>
  </action>
  <action name="action_win_connexion">
   <property name="text">
    <string>Afficher la fenêtre de connexion</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tarotv::ui::vue_jhj</class>
   <extends>QListView</extends>
   <header>vue_jhj.hpp</header>
   <slots>
    <signal>selection_changed()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_quitter</sender>
   <signal>triggered()</signal>
   <receiver>main_window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_win_listejoueurs</sender>
   <signal>triggered()</signal>
   <receiver>dock_liste_joueurs</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dock_liste_joueurs</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>action_win_listejoueurs</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_win_discussion</sender>
   <signal>triggered()</signal>
   <receiver>dock_discussion</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>487</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dock_discussion</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>action_win_discussion</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_win_connexion</sender>
   <signal>triggered()</signal>
   <receiver>dock_connexion</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dock_connexion</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>action_win_connexion</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>main_window</sender>
   <signal>server_ok(bool)</signal>
   <receiver>conteneur_identification</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>659</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>main_window</sender>
   <signal>auth_ok(bool)</signal>
   <receiver>bouton_connexion</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>538</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>main_window</sender>
   <signal>auth_ok(bool)</signal>
   <receiver>bouton_deconnexion</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>538</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bouton_changer_serveur</sender>
   <signal>toggled(bool)</signal>
   <receiver>champ_adresse</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>693</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>663</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>main_window</sender>
   <signal>server_ok(bool)</signal>
   <receiver>bouton_changer_serveur</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>687</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>716</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>main_window</sender>
   <signal>message(QString)</signal>
   <receiver>statusbar</receiver>
   <slot>showMessage(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>586</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>main_window</sender>
   <signal>auth_ok(bool)</signal>
   <receiver>champ_nom</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>479</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>auth_ok(bool)</signal>
  <signal>server_ok(bool)</signal>
  <signal>message(QString)</signal>
 </slots>
</ui>
